.editorconfig

```
@@ -42,6 +42,17 @@ indent_size = 4
 end_of_line = crlf
 charset = utf-8
 
+csharp_style_prefer_primary_constructors = false
+csharp_style_implicit_object_creation_when_type_is_apparent = false:silent
+csharp_style_var_when_type_is_apparent = true:silent
```

Actually we should probably align `var` and `new()` rules to always `false` (prefer fully spelled out) but silent.

We can make this a visible suggestion too, if that's prefered.

```suggestion
csharp_style_var_when_type_is_apparent = false:silent
```